student_registry/Gemfile

```diff
@@ -0,0 +1,56 @@
+source 'https://rubygems.org'
```

Comment:
You don't need to create your own rails project - edit the existing `session_2/student_registry` folder.

Reply:
Sure